scripts/tests/buildDSAv2.ts

```diff
@@ -0,0 +1,27 @@
+import { ethers } from "hardhat";
+
+import { addresses as addressesPolygon } from "./polygon/addresses";
+import { addresses } from "./mainnet/addresses";
+import { abis } from "../constant/abis";
+import { abi } from "../../deployements/mainnet/Implementation_m1.sol/InstaImplementationM1.json";
+
+function getAddress(network: string | undefined) {
+  if (network === "polygon") return addressesPolygon.core.instaIndex;
+  // else if (network === "arbitrum") return addressesPolygon.core.instaIndex;
+  // else if (network === "avalanche") return addressesPolygon.core.instaIndex;
```

Comment:
@pradyuman-verma Let's uncomment these and add avalanche and arbitrum constants?

Reply:
okay on it